### PR TITLE
Add example for XSLTProcessor `removeParameter()` method

### PR DESCRIPTION
**Pull request by T34-active**
### Description
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/XSLTProcessor/transformToFragment
### Related issues and pull requests
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/api/xsltprocessor/transformtofragment/index.md
* Last commit: https://github.com/mdn/content/commit/1df5bd15dd487615fbdef41e6c3e008403c7ae81

### CODE REVIEW

**Review by Josh-Cena — CHANGES_REQUESTED** [review]
Hi @T34-active, I've made some changes to your branch that are tricky to explain via comments, namely:

- Add a subheading called "Using \<method>()"
- Add H4 headings called "HTML", "JavaScript", and a new "Result" section that uses [`EmbedLiveSample`](https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Page_structures/Live_samples)
- Dedent the `xmlString`
- Remove comments for the setup part; only explain things that pertain to the page's topic
- Always have a visual comparison before&after calling the method

Could you apply the same change to your other PRs?

Also I could not get the comparison working. Could you make sure the example after `removeParameter` is displayed as intended?

**Comment by T34-active** [comment]
> Hi @T34-active, I've made some changes to your branch that are tricky to explain via comments, namely:
> 
> * Add a subheading called "Using ()"
> * Add H4 headings called "HTML", "JavaScript", and a new "Result" section that uses [`EmbedLiveSample`](https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Page_structures/Live_samples)
> * Dedent the `xmlString`
> * Remove comments for the setup part; only explain things that pertain to the page's topic
> * Always have a visual comparison before&after calling the method
> 
> Could you apply the same change to your other PRs?
> 
> Also I could not get the comparison working. Could you make sure the example after `removeParameter` is displayed as intended?

The document has been formatted as you requested, check out the formatting after I fixed it.

File: files/en-us/web/api/xsltprocessor/removeparameter/index.md [hunk 1]
```diff
@@ -29,6 +29,68 @@ None ({{jsxref("undefined")}}).
...
+`;
+
+const parser = new DOMParser();
+const xmlDoc = parser.parseFromString(xmlString, "application/xml");
+const xsltDoc = parser.parseFromString(xsltString, "application/xml");
+
+const xsltProcessor = new XSLTProcessor();
+xsltProcessor.importStylesheet(xsltDoc);
+
+xsltProcessor.setParameter(null, "showItems", "yes");
```
Comment (Josh-Cena) [inline files/en-us/web/api/xsltprocessor/removeparameter/index.md:76]: Same as `clearParameters`: set this to "no" and render something else

Comment (T34-active) [reply files/en-us/web/api/xsltprocessor/removeparameter/index.md:76]:
```
const xmlString = `
<items>
  <item>Item 1</item>
  <item>Item 2</item>
</items>
`;

const xsltString = `
<xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
  <xsl:param name="showItems" select="'yes'"/>
  <xsl:template match="/">
    <!-- If showItems is 'yes', display the list of items -->
    <xsl:if test="$showItems = 'yes'">
      <ul>
        <xsl:for-each select="items/item">
          <li><xsl:value-of select="."/></li>
        </xsl:for-each>
      </ul>
    </xsl:if>
    <!-- If showItems is 'no', display a message -->
    <xsl:if test="$showItems = 'no'">
      <div>No content to show</div>
    </xsl:if>
  </xsl:template>
</xsl:stylesheet>
`;

const parser = new DOMParser();
const xmlDoc = parser.parseFromString(xmlString, "application/xml");
const xsltDoc = parser.parseFromString(xsltString, "application/xml");

const xsltProcessor = new XSLTProcessor();
xsltProcessor.importStylesheet(xsltDoc);

// Set 'showItems' to 'yes' and perform the first transformation
xsltProcessor.setParameter(null, "showItems", "yes");
const resultContainer = document.getElementById("result");
let resultFragment = xsltProcessor.transformToFragment(xmlDoc, document);
resultContainer.appendChild(resultFragment);

// Add a horizontal rule to separate the results
resultContainer.appendChild(document.createElement("hr"));

// Remove the 'showItems' parameter, which will revert it to its default value
xsltProcessor.removeParameter(null, "showItems");
resultFragment = xsltProcessor.transformToFragment(xmlDoc, document);
resultContainer.appendChild(resultFragment);

// Set 'showItems' to 'no' and perform the third transformation
resultContainer.appendChild(document.createElement("hr"));
xsltProcessor.setParameter(null, "showItems", "no");
resultFragment = xsltProcessor.transformToFragment(xmlDoc, document);
resultContainer.appendChild(resultFragment);
```

Comment (Josh-Cena) [reply files/en-us/web/api/xsltprocessor/removeparameter/index.md:76]: Same idea: what if we just set it to "no" for the first `transformToFragment` and then remove it (restoring it to default "yes") for the second `transformToFragment`? No need to have three examples.

Comment (T34-active) [reply files/en-us/web/api/xsltprocessor/removeparameter/index.md:76]:
```js
const xmlString = `
<items>
  <item>Item 1</item>
  <item>Item 2</item>
</items>
`;

const xsltString = `
<xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
  <xsl:param name="showItems" select="'yes'"/>
  <xsl:template match="/">
    <!-- If showItems is 'yes', display the list of items -->
    <xsl:if test="$showItems = 'yes'">
      <ul>
        <xsl:for-each select="items/item">
          <li><xsl:value-of select="."/></li>
        </xsl:for-each>
      </ul>
    </xsl:if>
    <!-- If showItems is 'no', display a message -->
    <xsl:if test="$showItems = 'no'">
      <div>No content to show</div>
    </xsl:if>
  </xsl:template>
</xsl:stylesheet>
`;

const parser = new DOMParser();
const xmlDoc = parser.parseFromString(xmlString, "application/xml");
const xsltDoc = parser.parseFromString(xsltString, "application/xml");

const xsltProcessor = new XSLTProcessor();
xsltProcessor.importStylesheet(xsltDoc);

// Set 'showItems' to 'no' and perform the first transformation
xsltProcessor.setParameter(null, "showItems", "no");
const resultContainer = document.getElementById("result");
let resultFragment = xsltProcessor.transformToFragment(xmlDoc, document);
resultContainer.appendChild(resultFragment);

// Add a horizontal rule to separate the results
resultContainer.appendChild(document.createElement("hr"));

// Remove the 'showItems' parameter, reverting it to the default value ('yes')
xsltProcessor.removeParameter(null, "showItems");
resultFragment = xsltProcessor.transformToFragment(xmlDoc, document);
resultContainer.appendChild(resultFragment);

```

**Review by Josh-Cena — APPROVED** [review]
Looks good, thanks!

**Pull request merged**